src/OXRS_LCD.cpp

```diff
@@ -54,10 +47,18 @@ void OXRS_LCD::begin (uint32_t ontime_event, uint32_t ontime_display)
   
   _ontime_display = ontime_display;
   _ontime_event = ontime_event;
+
+  Serial.print(F("[file] mounting SPIFFS..."));
+  if (!SPIFFS.begin())
```

Comment:
I updated the Rack32 library to pass `.begin(true)` which will format if there is an error mounting - worth doing that here?

Reply:
I don't think so. I'm just reading from SPIFFS and formatting wouldn't help anything.
I leave the decission to format up to Rack32. Better to maintain if it's done in one place only.

I made the other suggestet changes and will commit and merge everything tomorrow after final testing.